testsuite/drivers/helpers.py

```diff
@@ -5,21 +5,6 @@
 import os
 
 
-# Check a file contains a concrete line
-def check_line_in(filename, line):
-    """
-    Assert that the `filename` text file contains at least one line that
-    contains `line`.
-    """
-    with open(filename, 'r') as f:
-        for l in f:
-            if l.rstrip() == line:
-                break
-        else:
-            assert False, 'Could not find {} in {}'.format(
-                repr(line), filename)
```

Comment:
Don't we want this one just in case for the future? Or is there something wrong with it?

Reply:
Ah yes, this should be in a separate commit, but this function is actually a duplicate. There is exactly the same one defined in the same file.